programme-schedule.md

```diff
@@ -0,0 +1,56 @@
+# Week 1: Induction
+
```

Comment:
@laurafitzgerald @JameelB yes, let's have a discussion on how we want to house mentor + student resources. I don't see an issue with housing both in the same repo, but we can discuss it. 

Plus we need to discuss the potential for using a tutors site for student content.

Also, let's decide if we want to push content so we can iterate on it quickly, or plan/ structure things a bit first